netci.groovy

```diff
@@ -1650,7 +1650,12 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
                     }
 
                     if (enableCorefxTesting) {
-                        buildOpts += ' skiptests';
+                        // We shouldn't need to build the tests. However, run-corefx-tests.py currently depends on having the restored corefx
+                        // package available, to determine the correct corefx version git commit hash, and we need to build the tests before
+                        // running "tests\\runtest.cmd GenerateLayoutOnly". So build the pri-0 tests to make this happen.
+                        //
+                        // buildOpts += ' skiptests';
```

Comment:
This seems like an expensive hack. To my knowledge runtest generateLayoutOnly does not require built tests to have run.

Reply:
Yes, it is (relatively) expensive. But you do actually need to have built tests (on Windows) before GenerateLayoutOnly will work. I don't think it used to be that way.